tango_ros_common/tango_ros_native/src/tango_ros_node.cpp

```diff
@@ -233,27 +220,115 @@ std::string toFrameId(const TangoCoordinateFrameType& tango_frame_type) {
   }
   return string_frame_type;
 }
+// Converts TangoCameraIntrinsics to sensor_msgs::CameraInfo.
+// @param camera_intrinsics, TangoCameraIntrinsics to convert.
+// @param camera_info, the output CameraInfo.
+void toCameraInfo(const TangoCameraIntrinsics& camera_intrinsics,
+                  sensor_msgs::CameraInfo* camera_info) {
```

Comment:
Would you please add a reference to make explicit why you are configuring the camera info from ROS sensor_msgs like this?